opts/certs_test.go

```diff
@@ -0,0 +1,59 @@
+package opts
+
+import (
+	"errors"
+	"testing"
+
+	"golang.org/x/sys/unix"
+)
+
+func TestDecodePem(t *testing.T) {
```

Comment:
What about MultiCert pem files? Test this, too, or is there no effect?

Reply:
Not even the qemu test runs with multi cert file :P